ja_jp/contacts-manage-contacts.adoc

```diff
@@ -13,7 +13,7 @@
 . 必要な情報を入力します。
 . btn:[保存] をクリックします。
 
-NOTE: 追加した連絡先は選択中のフォルダに格納されます。
+NOTE: 追加した連絡先は *連絡先* フォルダの下に保存されます。

```

Comment:
According to the comment from Dawood, it describes about Contacts tab.

Reply:
Fixed.